src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,55 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+    private boolean nullKeyExists;
+
+    public StorageImpl() {
+        keys = (K[]) new Object [DEFAULT_CAPACITY];
+        values = (V[]) new Object [DEFAULT_CAPACITY];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < DEFAULT_CAPACITY; i++) {
+            if ((keys[i] == key) || (keys[i] != null && keys[i].equals(key))) {
+                values[findIndex(keys, key)] = value;
```

Comment:
should be simplified
```suggestion
                values[i] = value;
```